frontend/packages/process-editor/src/components/Canvas/CanvasActionMenu/CanvasActionMenu.test.tsx

```diff
@@ -1,28 +1,44 @@
 import React from 'react';
-import { render as rtlRender, screen, act } from '@testing-library/react';
+import { render, screen, act } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import type { CanvasActionMenuProps } from './CanvasActionMenu';
 import { CanvasActionMenu } from './CanvasActionMenu';
 import { textMock } from '../../../../../../testing/mocks/i18nMock';
-import { BpmnContextProvider } from '../../../contexts/BpmnContext';
+import type { BpmnContextProps } from '../../../contexts/BpmnContext';
+import { BpmnContext } from '../../../contexts/BpmnContext';
 
 const mockBPMNXML: string = `<?xml version="1.0" encoding="UTF-8"?></xml>`;
 
 const mockAppLibVersion7: string = '7.0.0';
-const mockAppLibVersion8: string = '8.0.0';
 
 const mockOnSave = jest.fn();
 
 const defaultProps: CanvasActionMenuProps = {
   onSave: mockOnSave,
 };
 
+const defaultContextProps: BpmnContextProps = {
+  bpmnXml: mockBPMNXML,
+  modelerRef: null,
+  numberOfUnsavedChanges: 0,
+  setNumberOfUnsavedChanges: () => {},
+  dataTasksAdded: [],
+  setDataTasksAdded: () => {},
+  dataTasksRemoved: [],
+  setDataTasksRemoved: () => {},
+  getUpdatedXml: async () => '',
+  isEditAllowed: true,
+  appLibVersion: mockAppLibVersion7,
```

Comment:
nitpicking, but should the default be a valid version? Even though it does not affect the tests where it is irrelevant